3_javascript/javascript3/week1/Giphy/scripts.js

```diff
@@ -0,0 +1,53 @@
+domListener();
+function domListener() {
+  document.addEventListener("DOMContentLoaded", domLoader);
+}
+
+function domLoader() {
+  const inputSearch = document.getElementById("search");
+  const clickSearch = document.getElementById("click");
+  const numberOfSearch = document.getElementById("quantity");
+  const resultOfSearch = document.getElementById("result");
+
+  let toSearch = "";
+  let limitOfSearch = 25;
```

Comment:
`toSearch` sounds neither like a variable nor a function.... since it is the search parameter. keyword, it could be named accordingly to make it clearer! 
`limitOfSearch` is okay, but it could be clearer by being more concise - searchLimit. 
What you named those is absolutely fine, I am just doing what's called "nitpicking" in code reviews 😀